docs/src/solutions.md

```diff
@@ -249,6 +249,28 @@ lp_objective_perturbation_range
 lp_rhs_perturbation_range
 ```
 
+## Conflicts
+
+Sometimes, the model you input is infeasible, and some solvers can help you
+find the cause of this infeasibility by offering a conflict, i.e. a subset of
+the constraints that create this infeasibility.
+
+The function [`compute_conflict!`](@ref) is used to start the computation of
```

Comment:
CI seems to be failing, maybe use `JuMP.compute_conflict!`

Reply:
I just tried that, but it does not solve the problem… Also, it seems that the docs for `MOI.ConflictParticipationStatusCode` and `MOI.ConflictStatusCode` are not properly generated, unlike `MOI.TerminationStatusCode`. Is there any configuration missing?